libs/sql-schema-describer/tests/describers/postgres_describer_tests.rs

```diff
@@ -139,6 +139,7 @@ fn all_postgres_column_types_must_work(api: TestApi) {
                         0,
                     ),
                     name: "User",
+                    is_partition: false,
```

Comment:
These snapshots are causing a lot of pain :/

Reply:
(I know how annoying it is to update them all without missing one place / one database)

Reply:
I want to do something about that. The easiest thing I can think of right now is to just make updating them easier. Not sure how I want to do it yet, but I will try to come up with a plan these days. I'll start a slack thread about it.